tests/e2e/specs/out_of_game/home.spec.js

```diff
@@ -18,6 +18,26 @@ function assertSuccessfulJoin(gameState) {
   cy.contains('h1', `Lobby for ${gameState.name}`);
 }
 
+/*
+ * vueDevToolsScript is set in vue.config.js, so it's not feasible to test both the enabled
+ * and the disabled state without substantial refactoring in to how tests/env work. To that end
+ * it's easier to instead have 2 separate tests based on the ENABLE_VUE_DEVTOOLS value being
+ * passed to the cypress.config.js file on instantiation.
+ */
+describe.only('Vue Devtools', () => {
+  beforeEach(setup);
+
+  if (Cypress.env('ENABLE_VUE_DEVTOOLS') === 'true') {
+    it('Should include Vue Devtools when ENABLE_VUE_DEVTOOLS is true', () => {
+      cy.get('[data-vue-devtools]').should('have.length', 1);
+    });
+  } else {
+    it('Should not include Vue Devtools when ENABLE_VUE_DEVTOOLS is not true', () => {
+      cy.get('[data-vue-devtools]').should('have.length', 0);
+    });
+  }
+});
```

Comment:
Tried to explain as best as I could, but we still need to fork the logic in these tests due to the env being build time and related to how your index.js file is being generated.

That said, you can see the successful runs here:

| GUI | Client | Server |
|-----|-------|-------|
| ![Screen Shot 2022-06-22 at 3 44 29 PM](https://user-images.githubusercontent.com/414475/175125903-d4e77f12-e7a4-443b-b2da-2eb0e5d69497.png) | ![Screen Shot 2022-06-22 at 3 48 14 PM](https://user-images.githubusercontent.com/414475/175125976-2c431aec-88dd-490e-88bc-a9c399a980b8.png) |  ![Screen Shot 2022-06-22 at 3 46 42 PM](https://user-images.githubusercontent.com/414475/175125924-c96398b1-ff95-4cac-8ef0-ab3fba377ade.png) |